blaisebakundukize/Questioner
# #163251937 Remove dummy data and Build out REST Endpoints for Questioner

#### What does this PR do?
Have CRUD on questioner REST API endpoints
#### Description of Task to be completed?
Have the following endpoints working

```POST  /api/v1/meetups/```

```GET  /api/v1/meetups```

```GET  /api/v1/meetups/upcoming```

```GET  /api/v1/meetups/<meetupID>```

```POST  /api/v1/meetups/<meetupID>/rsvps```

```POST  /api/v1/meetups/<meetupID>questions```

```PATCH  /api/v1/questions/<questionID>/upvote```

```PATCH  /api/v1/questions/<questionID>downvote```
#### How should this be manually tested?
> - Ensure you have installed Nodejs which comes with npm
> - Navigate into the app root directory: `cd foldername`
> - Run `$ npm install`  from terminal to install all dependencies
> - Run tests to ensure the app is not broken: `npm test`
> - If test fail to run, install mocha globally: `npm install -g mocha`
> - Run tests again
> - To start the app run: `npm start`
> - If failed to start, install nodemon globally: `npm install -g nodemon`

#### What are the relevant pivotal tracker stories?
#163251937
